tests/staticfiles_tests/test_storage.py

```diff
@@ -77,6 +77,14 @@ def test_path_ignored_completely(self):
             self.assertIn(b"url()", content)
         self.assertPostCondition()
 
+    @override_settings(STATIC_URL="/")
+    def test_protocol_relative_url_ignored_with_static_url_set_to_slash(self):
+        relpath = "cached/css/ignored.55e7c226dda1.css"
```

Comment:
This test does not look like the others because the `hashed_file_path` helper above gives confusing output when `STATIC_URL` is set to `/`. How should I proceed?

1. Leave the test minimal like it is now
2. Assert the odd URL that is returned (`cachedcssignored.55e7c226dda1.css`)
3. Modify `hashed_file_path` to skip the `replace` if `STATIC_URL` is set to `/`

Reply:
> This test does not look like the others because the `hashed_file_path` helper above gives confusing output when `STATIC_URL` is set to `/`. How should I proceed?
> 
>     1. Leave the test minimal like it is now
> 
>     2. Assert the odd URL that is returned (`cachedcssignored.55e7c226dda1.css`)
> 
>     3. Modify `hashed_file_path` to skip the `replace` if `STATIC_URL` is set to `/`

I think we should modify `hashed_file_path` to use `.removeprefix(settings.STATIC_URL)` instead of `replace()`.

Reply:
Even better, thanks! Fixed in d6c3a52, and I updated the test in 5602c2c.